nephio-ansible-install/roles/gitea/create/tasks/main.yaml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Get UID
+  getent:
+    database: passwd
+  when:
+    - gitea_username is defined
+    - gitea_password is defined
+- name: Create a gitea directory
+  ansible.builtin.file:
+    path: /home/{{ cloud_user}}/gitea
+    state: directory
+    mode: '0755'
+- name: Start gitea container
+  docker_container:
+    name: gitea
+    image: gitea/gitea:1.17.3-rootless
+    networks:
+      - name: kind
+    user: "{{ getent_passwd[cloud_user].1 }}:{{ getent_passwd[cloud_user].2 }}"
+    restart: yes
+    ports:
+      - "127.0.0.1:3000:3000/tcp"
+      - "127.0.0.1:2222:2222/tcp"
+    env:
+        GITEA_APP_INI: "/var/lib/gitea/custom/conf/app.ini"
+        INSTALL_LOCK: "true"
+    volumes:
+      - /home/{{ cloud_user }}/gitea:/var/lib/gitea:rw
+  when:
+    - gitea_username is defined
+    - gitea_password is defined
+- name: Wait for gitea to initialise
+  ansible.builtin.wait_for:
+    path: /home/{{ cloud_user }}/gitea/ssh
+  when:
+    - gitea_username is defined
+    - gitea_password is defined
+- name: Creating nephio user in gitea
+  shell: |
+    docker exec -it gitea \
```

Comment:
I was able to execute the playbook without the above error once I removed the `-it` ... did anyone else have issue running the playbook as is?

Reply:
Hi @vjayaramrh , you are right. Removing the -t is doing the trick.
This would resolve the error message in your review comment above and also resolve the follow on error in your review comment below.

Reply:
@danaschwanden In my case, I removed the entire `-it`

Reply:
fixed [5c9d912](https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/5c9d912f784f71a0de58bbea88ccc54bbbbb9cfe)